ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.13.yaml

```diff
@@ -41,11 +41,11 @@ operator:
   substitutions:
   - pullspec: openshift.io/cert-manager-operator:latest
     with: pipeline:cert-manager-operator
-  - pullspec: quay.io/jetstack/cert-manager-controller:v1.12.4
```

Comment:
yes, we could not have mirrored v1.13.z due to reasons I cannot recall (there was some issue). Essentially, ended of using `quay.io/jetstack/cert-manager-*:v1.13.z` directly. 

If the tag replacements do not match CI falls back to the upstream source, instead of the CI mirrored image (both of which contents remain same, as we do not rebuild images from our downstream fork).